build.gradle

```diff
@@ -1,21 +1,19 @@
-plugins {
-    id 'java'
-    id 'org.jetbrains.intellij' version '0.3.5'
-}
 
 group 'io.vivy'
-version '1.0-SNAPSHOT'
-
-sourceCompatibility = 1.8
+version '1.0'
 
 repositories {
     mavenCentral()
 }
 
-dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-}
+subprojects {
+    apply plugin: 'java'
 
-intellij {
-    version '2018.2.1'
-}
+    dependencies {
+
```

Comment:
please also apply checkstyle to all the submodules

Reply:
done